box-share-sdk/src/main/java/com/box/androidsdk/share/activities/BoxInviteCollaboratorsActivity.java

```diff
@@ -67,9 +67,18 @@ public boolean onPrepareOptionsMenu(Menu menu) {
     }
 
     @Override
+    protected void onActivityResult(int requestCode, int resultCode, Intent data) {
```

Comment:
Earlier I was using the result but later I refactored it so that we update the list every time this activity is launched. I am thinking of just getting rid of this method and instead do something like:

```
// In the fragment
    @Override
    public void onResume() {
        super.onResume();
        refreshUi();
    }
```
So the fragment will refresh itself when it gets loaded.